src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -102,6 +102,9 @@
     @Column(name = "presentation_score")
     private Integer presentationScore;
 
+    @Column(name = "has_achievements", columnDefinition = "Boolean default false")
+    private Boolean hasAchievements = false; // default value
```

Comment:
Is this attribute set when the course is created by the instructor? If this is the case I would rename it to something like `achievementsEnabled`.

Reply:
Yes, it is. Renaming it then. 😊